code/modules/clothing/suits/utility.dm

```diff
@@ -128,6 +129,7 @@
 	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
 	resistance_flags = NONE
 	rad_flags = RAD_PROTECT_CONTENTS
+	snug_fit = TRUE

```

Comment:
Both of these are not a good idea, there are not many protections against bombs and rads, not a cool prank